warehouse/static/js/warehouse/controllers/localized_time_controller.js

```diff
@@ -49,6 +49,8 @@ export default class extends Controller {
       }
     }
 
+    const userTimezone = Intl.DateTimeFormat().resolvedOptions().timeZone;
+    isoDate = isoDate + ` (${userTimezone})`;
```

Comment:
This will work, but wouldn't just adding a "X" to the format string above be enough ? (https://date-fns.org/v2.21.3/docs/format)

Reply:
If I'm reading correctly those formatting options display UTC offsets in various ways, which while accurate... are not quite as human readable. Result from the proposed change reads like `America/New_York` for me.

Personally I'd prefer that (since it's identifiable to me as my timezone) over `-4 Z`

Reply:
Thanks for the review! I'll update the PR accordingly :+1:

Reply:
I'll also fix the breaking build. I didn't realize we do have tests for JS code (yaaaaaay)

Reply:
@ewdurbin and @ewjoachim I've updated the PR considering your suggestions. The only thing to notice is that we're kind of limited on format options with `date-fns` because Pypi is still using version `v1.19` and, thus, the "X" token is not available. 

But, from [1.19 docs](https://date-fns.org/v1.29.0/docs/format) I was able to use "Z" token instead.